index.js

```diff
@@ -2292,6 +2292,24 @@
                R.pipe(R[name], Just, R.filter(R.gte(_, 0))));
   };
 
+  //# append :: a -> Array a -> Array a
```

Comment:
Even with separate files we'd need to specify the readme order *somewhere*, but a list of filenames in the makefile would certainly be easier to read. ;)